package.json

```diff
@@ -330,6 +330,16 @@
         "title": "%commands.SAS.collapseAll%",
         "category": "SAS",
         "icon": "$(collapse-all)"
+      },
+      {
```

Comment:
It looks like not make sense for end user to call, right?
If so, how about somehow just export the function for other modules to call but not as a command?

Reply:
Let me explain why I did it this way.

In order to save state like this, we need the extension context, which is only available on extension creation. 

So either I save the extension context in some global access area, Or I create a command that gets and sets values like this.

It seemed to me that directly exposing the extension context was not a good thing to do.

Also, even though this is a command, it is not callable by the user, we are simply using vscodes command interface to enable actions like this.

Reply:
I've put my idea to
https://github.com/sassoftware/vscode-sas-extension/pull/183